### DISCUSSION TITLE
[Cinnamon] bluetooth icon

### DISCUSSION THREAD

**Discussion by JohnnyCalavera** [SpiralLinux/SpiralLinux-project#49] [General]
Not sure if we can do anything about it, but the bluetooth icon changes its look completely when disabled in SpiralLinux Cinnamon.
Linux Mint looks like this:
![Bildschirmfoto vom 2022-07-13 13-30-33](https://user-images.githubusercontent.com/16836649/178839537-327c51b2-5323-477f-84d4-b891f8517885.png) ![Bildschirmfoto vom 2022-07-13 13-30-07](https://user-images.githubusercontent.com/16836649/178839533-ce0c9a6b-7ebd-4290-b876-d6abe57ce08a.png) 

With SpiralLinux you get a blue symbol when you turn off bluetooth:
![Bildschirmfoto vom 2022-07-13 13-53-18](https://user-images.githubusercontent.com/16836649/178839854-e055d061-e23b-4191-a07e-548e25774ccd.png) ![Bildschirmfoto vom 2022-07-13 13-54-38](https://user-images.githubusercontent.com/16836649/178839865-050a01b5-617d-4b26-98bb-67bfd0986bbc.png)

Is there anything we could do to make the design more consistent?

**Comment by geckolinux** [comment]
Hi there, is Linux Mint using Blueman or the older Blueberry? Blueberry does have some hidden dconf settings for the tray icon style. Blueman unfortunately is much less polished than Blueberry, which has been deprecated and doesn't exist for Debian. At any rate, for Blueman I think you would have to change to a different icon theme. Or, you could make a local copy of the Faenza theme into `~/.themes/` and then swap out the `blueman-disabled.png` and/or `blueman-disabled.svg` (not sure which) for one with the same name from another theme. Then uninstall the Faenza theme from the system with Synaptic. Unfortunately this isn't something that I could include out of the box because I prefer to ship unmodified themes from the Debian repos with SpiralLinux.

**Comment by JohnnyCalavera** [comment]
It's Blueman because I tested the Linux Mint 21 Beta. I can completely understand that you prefer to ship unmodified themes from the Debian repos and I think that's what makes SpiralLinux so interesting - having a clean, well configured Debian. So I will play around with the themes and maybe things will look different when bookworm comes out next year. Thank you.
